eland/ml/pytorch/transformers.py

```diff
@@ -573,7 +573,13 @@ def _prepare_inputs(self) -> transformers.BatchEncoding:
 
 
 class TransformerModel:
-    def __init__(self, model_id: str, task_type: str, quantize: bool = False):
+    def __init__(
+        self,
+        model_id: str,
```

Comment:
Can we change this API to only accept keyword arguments? Then we can add new arguments whenever we want without worrying about positioning weirdness.

Reply:
++ I learned a new trick today